src/Makefile

```diff
@@ -40,7 +40,8 @@ BLST_BUILDSCRIPT_FLAGS = -D__BLST_PORTABLE__
 LIBS = $(BLST_LIBRARY)
 
 # List of individual implementation files.
-IMPL_FILES = common.c eip4844.c eip7594.c setup.c
+IMPL_FILES = common.c debug.c eip4844.c eip7594.c fft.c setup.c
```

Comment:
I guess a question for this PR is whether we want to introduce some directory structure like:
- `src/ckzg.c`
- `src/setup.c`
- `src/eip4844/eip4844.c`
- `src/eip7594/eip7594.c`
- `src/eip7594/fft.c`

So that we don't dump all the files on the root dir. I think I like the structure I suggest above, but no strong opinion. AFAIU it should not complicate the building process.

Reply:
This is a good idea. We should definitely do this.

Reply:
Let's do this in a separate PR.